lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2793,6 +2796,8 @@ ParseResult FIRStmtParser::parseLeadingExpStmt(Value lhs) {
 
   locationProcessor.setLoc(loc);
 
+  auto lhsType = lhs.getType().cast<FIRRTLType>();
+  auto rhsType = rhs.getType().cast<FIRRTLType>();
   auto lhsPType = lhs.getType().cast<FIRRTLType>().getPassiveType();
   auto rhsPType = rhs.getType().cast<FIRRTLType>().getPassiveType();
```

Comment:
You could update `lhsPType`/`rhsPTtype` to use these new `lhsType`/`rhsType`.

Reply:
Good call, thanks.  Think I meant to drop those entirely (since they're only used in disabled code that follows), but decided that can happen separately/later :).  Will push fix shortly.

Reply:
(dropped this code)